src/services/DfusionService.ts

```diff
@@ -0,0 +1,243 @@
+import Web3 from 'web3'
+
+import Logger from 'helpers/Logger'
+import { ContractEventLog } from 'contracts/types'
+import { OrderPlacement, StablecoinConverter } from 'contracts/StablecoinConverter'
+import packageJson from '../../package.json'
+import { BigNumber } from 'bignumber.js/bignumber.js'
```

Comment:
Why such strange import?

Reply:
Why BigNumber not BN? You intend to do division?

Reply:
> Why such strange import?

It was not working, I saw that weird import in stackoverflow or something, it worked. I tried to remove it now, it worked still, so no idea what happened the first time!

> Why BigNumber not BN? You intend to do division?

yes, is for that